exporters/webhook/models/pelorus_webhook.py

```diff
@@ -42,14 +42,15 @@ class PelorusPayload(BaseModel):
 
     Attributes:
         app (str): Application name.
-        timestamp (str): Timestamp of the event. This is different from the
-                         time when the webhook could have been received.
+        timestamp (str): 10 digit EPOCH timestamp of the event. This
+                         is different from the time when the webhook
+                         could have been received.
     """
 
     # Even if we consider git project name as app, it still should be below 100
     app: str = Field(max_length=200)
-    # ISO 8601 is 19 chars long, so don't expect any format can be longer then 50 chars
-    timestamp: str = Field(max_length=50)
+
+    timestamp: str = Field(regex=r"^\d{10}$")  # 10 digit number
```

Comment:
using `int` with `gt` and `lt` wouldn't do the trick as well? Just so we do not use regex

Reply:
Yes, but we allow this to be a string of 10 digits as well. What's wrong with regex?

Reply:
I think if is int, it accepts both 3 and "3"

We need to interpret it (in this case that is small, no big trouble)

Reply:
It's same vice-versa. I know it's not correct to think about possible future implementation, but we may want to change this to consume other types of timestamp formats adding other validators and then this will be required to be str.

Reply:
Tested with:
```python
    timestamp: int = Field(ge=1577840461, le=2524611661) # between 2020 and 2050.
```

Got the error on one of the tests:
```shell

        query_result = REGISTRY.get_sample_value(
            "test_committime_metrics",
            labels=metric_labels,
        )

>       assert query_result == timestamp
E       AssertionError: assert 1234567890 == '1234567890'

exporters/tests/test_in_memory_metric.py:108: AssertionError
```

The prometheus is returning str, while we pass int. We could have this, but still prefer str over int as we do str(int) at some point when storing data to prometheus anyway, so why not to accept this from payload.

Reply:
Actually it should be:

    timestamp: int = Field(None, ge=1577840461, le=2524611661) # between 2020 and 2050.

Otherwise the ge and le is not taken into account that's we have not allowed '1234567890', but still str over int in my opinion.

Reply:
Ok agree to change to int, if needed we can always switch back to str. Having int added some date constrains to get some reasonable date validation. Think the 2060 as the gt limit year is fine at this point (thinking of  Y2K bug... but I am pretty sure this data won't have lifespan of 35+ years from now).